README.md

```diff
@@ -21,6 +21,9 @@
 <p align="center">
 	<a href="https://join.slack.com/t/superduperdb/shared_invite/zt-1zuojj0k0-RjAYBs1TDsvEa7yaFGa6QA">
     <img src="https://img.shields.io/badge/slack-superduperdb-8A2BE2?logo=slack" alt="slack">
+    </a>
+    <a  href="https://twitter.com/superduperdb">
+<img  src="https://img.shields.io/badge/Twitter/X - Follow superduperdb-1DA1F2?logo=twitter"  alt="Twitter">
```

Comment:
@Ayush9026 Thanks for the contribution, 
Is it possible to make the badge just 
`Twitter`

Reply:
@thejumpman2323 Yes it is possible to make the badge just Twitter i will raise PR.

Reply:
@Ayush9026 you have to change it in the same pr!

Reply:
@thejumpman2323 okay i will change in the same pr.